src/main/java/Duke.java

```diff
@@ -1,10 +1,303 @@
+import java.util.Scanner;
+
 public class Duke {
+
+    public static final int MAX_NUMBER_OF_TASKS = 100;
+    public static final int TASK_ARGUMENT = 1;
+    public static final String TASK_SEPARATOR = " ";
+    public static final String DEADLINE_SEPARATOR = "/by ";
+    public static final String EVENT_SEPARATOR = "/at ";
+    public static final String TASK_PADDING = "   ";
+    public static final String NUMBER_LIST_SEPARATOR = ". ";
+    public static final String LIST_COMMAND = "list";
+    public static final String ADD_TODO_COMMAND = "todo ";
+    public static final String ADD_DEADLINE_COMMAND = "deadline ";
+    public static final String ADD_EVENT_COMMAND = "event ";
+    public static final String MARK_AS_DONE_COMMAND = "done ";
+    public static final String HELP_COMMAND = "help";
+    public static final String GREETING_COMMAND = "hello";
+    public static final String EXIT_COMMAND_1 = "bye";
+    public static final String EXIT_COMMAND_2 = "exit";
```

Comment:
Good job in refactoring the constants for better readability, however you should avoid naming constants with similar names like EXIT_COMMAND_1 and EXIT_COMMAND_2 as it could be confusing for other people. You can try using COMMAND_EXIT and COMMAND_BYE.